src/style.css

```diff
@@ -1,3 +1,68 @@
-body {
+*,
+*::before,
+*::after {
   margin: 0;
+  padding: 0;
+}
```

Comment:
it is bad idea to reset margin/padding for all elements in page

Reply:
Why? What a reason for it? What alternative and why it will be better?